packages/react-tinacms-github/src/github-client/index.ts

```diff
@@ -73,7 +72,18 @@ export class GithubClient {
     })
   }
 
-  async fetchExistingPR(forkRepoFullName: string, headBranch: string) {
+  get repoFullName(): string {
+    return getForkName() || this.baseRepoFullName
```

Comment:
I could see this maybe affecting behaviour in the short-term (If your fork cookie gets deleted, we might start committing to the base repo)

Reply:
Very true, I'll make it throw instead